### PR TITLE
Support to use SSH as webterminal only

### PR DESCRIPTION
**Pull request by pvizeli**
Fix: #1116 

recommended by @balloob

### CODE REVIEW

File: ssh/README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Follow these steps to get the add-on installed on your system:
 
 ## How to use
 
+You need enable the port for external access. You can just enter 22 as value or any other
```
Comment (rct) [inline ssh/README.md]: IMHO This doc change isn't helpful to a large number of people.   It just doesn't make sense unless the reader knows exactly what you are talking about. This was a breaking change. See new issue #1125.

File: ssh/config.json [hunk 2]
```diff
@@ -14,7 +15,7 @@
   "hassio_role": "manager",
   "auto_uart": true,
   "ports": {
-    "22/tcp": 22
+    "22/tcp": null
```
Comment (rct) [inline ssh/config.json]: IMHO this change should have been a separate change that was advertised as a breaking change.   It should also have been described as "Disable network ssh access by default".

File: ssh/config.json [hunk 3]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "SSH server",
-  "version": "8.3",
+  "name": "Terminal & SSH",
+  "version": "8.4",
   "slug": "ssh",
   "description": "Allow logging in remotely to Home Assistant using SSH",
```
Comment (rct) [inline ssh/config.json]: This description is no longer accurate.  By default the add-on no longer **allows logging in remotely using ssh**

File: ssh/README.md (hunk 1 above)
Comment (frenck) [reply ssh/README.md]: You've opened up an issue, not sure why you would create a comment here as well.

Please don't dig up closed PR's. Thanks.

File: ssh/config.json (hunk 2 above)
Comment (frenck) [reply ssh/config.json]: Please use issues for discussions and bug reports instead, thanks 👍

File: ssh/config.json (hunk 3 above)
Comment (frenck) [reply ssh/config.json]: Feel free to open up a PR to improve a thing, or open up an issue. Let's not use merged PR's for that, as it is not trackable. Thanks 👍

**Pull request merged**